scripts/ingests/bones_archive/ingest_bones_sources.py

```diff
@@ -90,52 +82,49 @@ def extractADS(link):
                 )  # ingest new names while were at it
                 names_ingested += 1
             except AstroDBError as e:
-                None  # only error is if there is a preexisting name anyways.
+                raise e  # only error is if there is a preexisting name anyways.
 
-    if match == None:
-         match = find_source_in_db(
+    if match is None:
+        match = find_source_in_db(
             db,
             source["NAME"],
             ra=source["RA"],
             dec=source["DEC"],
+            ra_col_name="ra",
+            dec_col_name="dec",
         )
-        
-      
 
     if len(match) == 0:
-        #ingest_publications for the ADS link
+        # ingest_publications for the ADS link
         ads = extractADS(source["ADS_Link"])
         adsMatch = None
         adsRef = source["Discovery Ref."]
         adsMatch = find_publication(
             db,
             bibcode = ads
         )
+        logger.debug(f"adsMatch: {adsMatch}")
 
-        if adsMatch[0] == False:
+        if not adsMatch[0]:
+            logger.debug(f"ingesting publication {ads}")
             ingest_publication(
                 db,
-                bibcode = ads,
-                reference = adsRef
+                bibcode=ads,
```

Comment:
THIS is the change that "fixes" your reference formatting problem.  you were specifying reference to be adsRef rather than letting the function figure it out itself.